caos.ansible_roles/roles/infra_agent/tasks/host/host-install-Darwin.yaml

```diff
@@ -34,9 +39,14 @@
 
 - name: Install agent
   become: false
-  command: "{{ brew_path }}/bin/brew reinstall --formula --build-from-source /Users/{{ ansible_user }}/newrelic-infra-agent.rb"
+  command: "{{ brew_path }}/bin/brew reinstall --force --formula --build-from-source /Users/{{ ansible_user }}/newrelic-infra-agent.rb"
+
+# Need to download https://formulae.brew.sh/api/formula.jws.json but cannot as root! Run `brew update` without `sudo` first then try again.
```

Comment:
Why is this 👇 being executed again?

Reply:
I think after installing, the infra-agent was showing the "update" message, but most probably a leftover. Let me remove it: https://github.com/newrelic-experimental/caos-ansible-roles/pull/68/commits/b197b61fcb352779c086dad9c9795a37b9aedb4e